docs/user-guide/migrate-to-greptimedb/migrate-from-influxdb.md

```diff
@@ -190,3 +190,5 @@ done
 </div>
 
 </DocTemplate>
+
+If you need a more detailed migration plan or example scripts, please provide the specific table structure and data volume. The [GreptimeDB official community](https://github.com/orgs/GreptimeTeam/discussions) will offer further support. Welcome to join the [Greptime Slack](http://greptime.com/slack).
```

Comment:
please move this line to `docs/db-cloud-shared/migrate/migrate-from-influxdb.md`